data/Vaaran Bhai Gurdas Ji/37.json

```diff
@@ -3668,7 +3668,7 @@
         "id": "BRGB",
         "source_page": 37,
         "source_line": 19,
-        "gurmukhi": "mnmuK AMnHy rwiq idhuM; suriq ivhUxy ckI Joˆdy [",
+        "gurmukhi": "mnmuK AMnHy rwiq idhuˆ; suriq ivhUxy ckI Joˆdy [",
```

Comment:
`JoNdy`

Reply:
This should be automatically done by a bot or caught by test somehow -- regardless, I did it manually for now